.multi-thread.yml

```diff
@@ -0,0 +1,33 @@
+trigger:
+  batch: true
+  branches:
+    include:
+      - main
+
+pr:
+  branches:
+    include:
+      - main
+  paths:
+    include:
+      - "*"
+
+resources:
+  containers:
+    - container: sgx
+      image: ccfciteam/ccf-ci:oe0.13.0-xxd
+      options: --publish-all --device /dev/sgx:/dev/sgx -v /dev/shm:/tmp/ccache
+
+jobs:
+  - template: .azure-pipelines-templates/common.yml
+    parameters:
+      target: SGX
+      env:
+        container: sgx
+        pool: Ubuntu-1804-DC8_v2
+      cmake_args: "-DCOMPILE_TARGETS=sgx -DWORKER_THREADS=2"
+      suffix: "MultiThread"
+      artifact_name: "MultiThread"
+      ctest_filter: '-LE "perf"'
```

Comment:
Excluding perf for now since the perf infra doesn't run in docker and the perf numbers are not correct with multi-threading (only one client is spawned).